spec/models/form_profile_spec.rb

```diff
@@ -866,6 +866,48 @@
     }
   end
 
+  let(:v21_22_expected_) do
+    {
+      'personalInformation' => {
+        'first' => user.first_name&.capitalize,
+        'middle' => user.middle_name&.capitalize,
+        'last' => user.last_name&.capitalize,
+        'suffix' => user.suffix,
+        'preferredName' => 'SAM',
+        'dateOfBirth' => user.birth_date,
+        'socialSecurityNumber' => user.ssn,
+        'serviceNumber' => '123455678'
+      },
+      'contactInformation' => {
+        'email' => user.pciu_email,
+        'phone' => us_phone,
+        'address' => address
+      },
+      'veteranServiceInformation' => veteran_service_information
+    }
+  end
+
+  let(:v21_22a_expected_) do
```

Comment:
Does any test call these expected blocks? I think you'll need to write a test for them, or add these forms to other tests like this: https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/models/form_profile_spec.rb#L1605-L1628

Also, I noticed there was an underscore at the end of the `expected` block names. Is this intentional?

Reply:
Holden and I have the same thoughts today 😃

Reply:
Added 21_22 and 21_22A to the existing prefill test [here](https://github.com/department-of-veterans-affairs/vets-api/pull/17598/commits/a514c4cbd70a5e8fcb6f1e27447352885fe80d8e).